### DISCUSSION TITLE
Increasing the number of precursor for quant

### DISCUSSION THREAD

**Discussion by johnpleid** [vdemichev/DiaNN#397] [Q&A]
Hi to everybody
does it exist a command line to increase the number of peptides used for the quantification instead of the Top 1
thank you
Giovanni

**Comment by vdemichev** [comment]
Hi Giovanni, can use --top [N], but in general it's recommended not to rely on these Top N quantities, but rather use the MaxLFQ quantities calculated by DIA-NN.

**Reply by jmsanfrutos** [reply]
Hi to everyone,

I have tried to use this command but it does not work, basically it does not start the search. I have used other command in the "Additional options" and no problems so far, just with this one. I do not get any error bar, it just stop the search.

Thanks

Julia

**Reply by jmsanfrutos** [reply]
Hi again,

I think I just realized which was the problem, instead of writing --top 3 I was writing --top [3].

Thanks 

Julia
